server/models/handlers/appointment.js

```diff
@@ -48,8 +48,35 @@ async function cancelWorkerAppointments(workerId, specificDate) {
     }
 }
 
+async function cancelSpecificAppointment(appointmentId) {
+    // If this method is somehow called without specifying values for the required parameters, false is returned.
+    if (_.isNil(appointmentId)) return false;
+    try {
+        const queryOutput = await appointmentModel.returnAppointment(appointmentId);
```

Comment:
Hi Peter,
For more clear variable naming, I would suggest adjusting the variable name to be `appointment` rather than `queryOutput` so it is more clear what this variable holds.

Reply:
Renamed!